dask_ml/model_selection/_incremental.py

```diff
@@ -33,35 +33,40 @@ def _partial_fit(model_and_meta, X, y, fit_params):
         A new dictionary with updated information.
     """
     with log_errors():
+        start = time()
         model, meta = model_and_meta
 
         if len(X):
             model = deepcopy(model)
             model.partial_fit(X, y, **(fit_params or {}))
 
         meta = dict(meta)
-        meta['time_step'] += 1
+        meta["partial_fit_calls"] += 1
+        meta["partial_fit_time"] = time() - start
 
         return model, meta
 
 
 def _score(model_and_meta, X, y, scorer):
+    start = time()
     model, meta = model_and_meta
     if scorer:
         score = scorer(model, X, y)
     else:
         score = model.score(X, y)
 
     meta = dict(meta)
-    meta.update(score=score)
+    meta.update(score=score, score_time=time() - start)
     return meta
 
 
 def _create_model(model, ident, **params):
     """ Create a model by cloning and then setting params """
     with log_errors(pdb=True):
         model = clone(model).set_params(**params)
-        return model, {'ident': ident, 'params': params, 'time_step': -1}
+        if hasattr(model, "initialize") and callable(model.initialize):
+            model.initialize()
```

Comment:
What is this for?

Reply:
This is for [skorch], and it's required. I tried adding it to the end of `__init__`, but then I think some hyperparameters couldn't change.

[skorch]:https://skorch.readthedocs.io/en/latest/

Reply:
We might not want to hack in something just for skorch.  This might have unintended consequences with other libraries.  This probably requires some thought.  Is this something that could be changed upstream?

Reply:
At the very least we'll need to protect this somewhat in case someone else provides an estimator that has an `initialize` method.  Perhaps we check if `"skorch" in type(est).__name__` or something.  We should also at least raise an issue upstream.